### DISCUSSION TITLE
外服水月肉鴿問題

### DISCUSSION THREAD

**Discussion by qwer753159456** [MaaAssistantArknights/MaaAssistantArknights#4907] [Q&A]
您好 想請問maa外服的水月肉鴿會一直無限循環在主頁面以及肉鴿界面是什麼原因呢?
肉鴿裡的設定基本上能調的都已經調了 想請問是外服目前還不適配嗎?

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/4897

**Reply by qwer753159456** [reply]
原來已經有討論串了 剛才搜了一下沒有看到 十分抱歉
